src/evidently/collector/app.py

```diff
@@ -46,14 +46,12 @@
 logger = logging.getLogger(__name__)
 
 
-@post(
-    "/{id:str}",
-)
-async def create_collector(
+@post("/{id:str}", sync_to_thread=True)
+def create_collector(
     id: Annotated[str, Parameter(description="Collector ID")],
     parsed_json: CollectorConfig,
-    service: CollectorServiceConfig,
-    storage: CollectorStorage,
+    service: Annotated[CollectorServiceConfig, Dependency(skip_validation=True)],
+    storage: Annotated[CollectorStorage, Dependency(skip_validation=True)],
```

Comment:
I'm not entirely sure why these are Pydantic models, as they don't seem to be serialized/validated anywhere. 

Them being Pydantic models however, and injected via Litestar's DI, means that they'll get validated on every call, which, for complex Pydantic objects, especially when using Pydantic 1, incurs a not insignificant performance overhead. Using `skip_validation=True` prevents this validation from taking place.